spec/parsers/generic_spec.rb

```diff
@@ -1,30 +0,0 @@
-require 'spec_helper'
-
-describe Bibliothecary::Parsers::Generic do
-  it 'has a platform name' do
-    expect(described_class.platform_name).to eq('generic')
-  end
-
-  it 'parses dependencies from dependencies.csv' do
-    expect(described_class.analyse_contents('dependencies.csv', load_fixture('dependencies.csv'))).to eq({
```

Comment:
is the `dependencies.csv` fixture no longer used, or even now an inaccurate example?